components/collective-page/sections/Budget.js

```diff
@@ -36,11 +36,13 @@ export const budgetSectionWithHostQuery = gqlV2/* GraphQL */ `
       limit: $limit
       hasExpense: false
       kind: $kind
+      includeIncognitoTransactions: true
       includeGiftCardTransactions: true
+      includeChildrenTransactions: true
     ) {
       ...TransactionsQueryCollectionFragment
     }
-    expenses(account: { slug: $slug }, limit: $limit) {
+    expenses(account: { slug: $slug }, limit: $limit, includeChildrenExpenses: true) {
```

Comment:
Not sure if we should include children's transactions on the collective page by default.

Reply:
The approach is to have it by default everywhere for consistency. Unless there is a huge blocker, let's see how it goes.